templates/terraform/aws/eks/eks_cluster/main.tf

```diff
@@ -42,40 +42,49 @@ locals {
 
   node_group_name = "subspace-default"
 
+  #---------------------------------------------------------------
   # ARGOCD ADD-ON APPLICATION
+  #---------------------------------------------------------------
 
   aws_addons = {
-    enable_cert_manager                          = true
-    enable_aws_ebs_csi_driver                    = true
-    enable_aws_efs_csi_driver                    = true
-    enable_aws_cloudwatch_metrics                = true
-    enable_aws_privateca_issuer                  = true
-    enable_cluster_autoscaler                    = true
-    enable_external_dns                          = true
-    enable_external_secrets                      = true
-    enable_aws_load_balancer_controller          = true
-    enable_aws_for_fluentbit                     = true
-    enable_aws_node_termination_handler          = true
-    enable_karpenter                             = true
-    enable_velero                                = true
-    enable_aws_gateway_api_controller            = true
-    enable_aws_secrets_store_csi_driver_provider = true
+    enable_cert_manager          = true
+    enable_aws_ebs_csi_resources = true # generate gp2 and gp3 storage classes for ebs-csi
+    #enable_aws_efs_csi_driver                    = true
+    #enable_aws_fsx_csi_driver                    = true
+    enable_aws_cloudwatch_metrics = true
+    #enable_aws_privateca_issuer                  = true
+    #enable_cluster_autoscaler                    = true
+    enable_external_dns                 = true
+    enable_external_secrets             = true
+    enable_aws_load_balancer_controller = true
+    #enable_fargate_fluentbit                     = true
+    enable_aws_for_fluentbit = true
+    #enable_aws_node_termination_handler          = true
+    enable_karpenter = true
+    #enable_velero                                = true
```

Comment:
why keep them if we dont need them ?

Reply:
I've removed the ones that we will never use, but commented ones that we will or likely use later in the cluster.